.gitignore

```diff
@@ -19,6 +22,7 @@ dist/
 __pycache__/
 htmlcov/
 *.egg-info/
+doc/**.ipynb
```

Comment:
Okay fair enough ^^

Reply:
I thought that was what you meant 😃

Reply:
You can have 'hierarchical' `.gitignore` files as well, with relatives paths. So, here, `doc/.gitignore`. Doesn't matter, effect is the same, only now if you would move/rename the `doc` folder (unlikely), you would have to modify the `.gitignore` in the root folder.